packages/amazonq/src/extensionWeb.ts

```diff
@@ -0,0 +1,15 @@
+/*!
```

Comment:
off topic / not a blocker: should we rename this to main.ts/mainWeb.ts as in toolkit: https://github.com/aws/aws-toolkit-vscode/blob/d27031ea5f26ece189a0921616e4676edbc84027/packages/toolkit/src/main.ts

or vice versa?

Reply:
I agree, I was thinking that we should go back to using `extension`. I'll look to update this